docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md

```diff
@@ -103,7 +103,7 @@ The following tables list permissions associated with basic and fixed roles.
 
 ### Alerting roles
 
-If alerting is [enabled]({{< relref "../../../../alerting/set-up/migrating-alerts/opt-out/" >}}), you can use predefined roles to manage user access to alert rules, alert instances, and alert notification settings and create custom roles to limit user access to alert rules in a folder.
+If alerting is [enabled]({{< relref "../../../../alerting/set-up/migrating-alerts/" >}}), you can use predefined roles to manage user access to alert rules, alert instances, and alert notification settings and create custom roles to limit user access to alert rules in a folder.
```

Comment:
Arguments to the `relref` shortcode shouldn't have the trailing slash as it can break resolution (https://grafana.com/docs/writers-toolkit/review/doc-validator/errata/#relref-has-trailing-slash).